client/src/containers/Dispositif/Dispositif.js

```diff
@@ -172,6 +172,25 @@ export class Dispositif extends Component {
     this.checkUserFetchedAndInitialize();
     window.scrollTo(0, 0);
     // this._initializeDispositif(this.props);
+    const menu = document.querySelector(".left-side-col");
+    const content = document.querySelector(".col-middle");
+    let posYNav = null;
+    if (menu && content && !isMobile) {
+      posYNav = menu.offsetTop;
+      window.addEventListener("scroll", () => {
```

Comment:
ici il faut que tu mettes 
window.addEventListener("scroll", this.handleScroll)

dans le componentWillUnmount tu mettes
window.removeEventListener("scroll", this.handleScroll)

ensuite tu crees une fonction handleScroll qui contient ta fonction qui modifie le style

Reply:
tu peux prendre exemple sur BandeauEdition, cette logique est implémentée

Reply:
ca evitera de dupliquer ta fonction dans le componentWillMount et WillUnMount